### DISCUSSION TITLE
Question on where Osmand files are stored in Android

### DISCUSSION THREAD

**Discussion by JohnJ80** [osmandapp/OsmAnd#12194] [Q&A]
I have been using Osmand on my bike computer (Hammerhead Karoo) that is android based.  I have written some shell scripts for the ADB tools to backup and restore the applications and their data that I have loaded onto the bike computer.  

When I restore Osmand, I get all the maps, routes/tracks, favorites etc... into the right places but the map, application, and screen preferences are not in the right place and they have to be set up all over again.  So my thought is that I'm not properly copying these files and then restoring them.  

Is there information on where all of this is stored and which files/directories contain what data?

Thanks.

**Comment by mariush444** [comment]
it seems that profile can be exported/imported only.
on rooted phone it is probably /data/data/net.osmand.plus/shared_prefs/*.xml
